### DISCUSSION TITLE
Need some help on running hass-emulated-hue

### DISCUSSION THREAD

**Discussion by giosem** [hass-emulated-hue/core#263] [General]
Hello,
I'm trying to run hass-emulated-hue with docker-compose, I'm not able to start it.

By the way, I've been able to start the docker with the command: 
docker run -t -d --network=host -e HASS_TOKEN=abc -e HASS_URL=123 ghcr.io/hass-emulated-hue/core:0.2.0

Everything works fine, no errors. 

If I go to: http://myIP/api/pi/lights  I get:
[{"error": {"address": "/lights", "description": "unauthorized user", "type": 1}}]

If I go to: http://myIP/description.xml I get:
<root xmlns="urn:schemas-upnp-org:device-1-0">
<specVersion>
<major>1</major>
<minor>0</minor>
</specVersion>
<URLBase>http://myIP:80/</URLBase>
<device>
<deviceType>urn:schemas-upnp-org:device:Basic:1</deviceType>
<friendlyName>Home Assistant (myIP)</friendlyName>
<manufacturer>Signify</manufacturer>
<manufacturerURL>http://www.philips-hue.com</manufacturerURL>
<modelDescription>Philips hue Personal Wireless Lighting</modelDescription>
<modelName>Philips hue bridge 2015</modelName>
<modelNumber>BSB002</modelNumber>
<modelURL>http://www.philips-hue.com</modelURL>
<serialNumber>b827eb77a19f</serialNumber>
<UDN>uuid:2f402f80-da50-11e1-9b23-b827eb77a19f</UDN>
<presentationURL>index.html</presentationURL>
<iconList>
<icon>
<mimetype>image/png</mimetype>
<height>48</height>
<width>48</width>
<depth>24</depth>
<url>hue_logo_0.png</url>
</icon>
</iconList>
</device>
</root>

Can someone help me understanding how to fix that "unauthorized user" message above?

Thanks
Giosemmo

**Comment by alexyao2015** [comment]
It's working correctly. You're just running into standard authentication errors with the hue API. If you're specifically developing against it, you need to authenticate using a token.

**Reply by giosem** [reply]
Thanks @alexyao2015! 
I'm not developing, I'm just trying to configure hass-emulated-hue to use Alexa app and voice command to switch lights on and off.

It seems I'm not able to get this working.

Do you have any suggestion on how to set this up or troubleshoot?

It will be very helpful! I feel like I'm almost done, but something is still missing somewhere. :)

Thank you
Bye
Giosemmo

**Reply by alexyao2015** [reply]
I'm honestly not sure if it works with Alexa or not (don't own one personally). If you're trying to use Alexa, however, I would suggest taking a look at the built in functionality within home assistant to connect directly to Alexa.
